### ISSUE TITLE
USB Accessory for Home Assistant on umbrel 

### ISSUE THREAD

**Issue by jonnylangefeld** [getumbrel/umbrel-apps#581]

**Comment by evercape** [comment]
When I run lsusb I get the following:

`umbrel@umbrel:~ $ lsusb
Bus 002 Device 002: ID 2109:0715 VIA Labs, Inc. 
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub`

Here is an older link how someone added the device to the docker-compose.yaml in HASS however if umbrel OS does not even permit USB devices other than the main HD then this is an issue?

**Comment by evercape** [comment]
Me again: https://apps.umbrel.com/app/home-assistant

Apparently USB devices are now discovered:

> This update brings Home Assistant to version 2023.7.1, **and now includes USB Discovery**! The previous update included automatic device discovery for devices connected to the same network, and the ability to save your personalized settings and automations.

Really odd, not working for me. I cannot find the Sonoff USB 3 device.

**Comment by CapGus** [comment]
this dongle error still happening?

**Comment by evercape** [comment]
Yes. I moved away to dedicated NUC, HA in docker on umbrel just not reliant enough I am afraid.

**Comment by dadofsambonzuki** [comment]
I couldn't get this working either. USB (Sonoff USB 3) not recognised.

**Comment by iammac2** [comment]
Hi did anyone get any further with USB devices for Home Assistant on Umbrel? Can see my Zooz Z-Wave stick on ‘lsusb’ output but HA doesn’t detect it. Reading some of the documents it sounds like I would need to run a Z-Wave JS Server and give Home Assistant the URL to the server - Umbrel provides Portainer but I’m not sure about the drive mappings 

This was my reference:
https://www.homeautomationguy.io/blog/docker-tips/installing-z-wave-js-with-docker-and-home-assistant

**Comment by jimsbr** [comment]
see below

**Comment by al-lac** [comment]
This should actually work just fine now:

<img width="1958" height="644" alt="Image" src="https://github.com/user-attachments/assets/c4febcc1-977f-4282-9455-97d8ab8116d4" />

Closing this, in case anyone has an issue, please just respond here, then I can re-open and look into it again.
